css/theme-blue.css

```diff
@@ -1,3 +1,28 @@
+table {
+    margin-top: 20px;
+    width: 100%;
+    background-color: transparent;
+    border-spacing: 0;
+    border-collapse: collapse;
+}
+
+table thead tr {
+    border-bottom: 2px solid #46a417 !important;
+    border: none;
+    background: #ffffff;
+    color: #46a417;
+}
+
+table td {
+    border-right: 2px solid #FFFFFF;
+    border-bottom: 2px solid #FFFFFF;
+    border-left: 2px solid #FFFFFF;
+    background: #e8e7e5;
+    color: #142848;
+    font-family: 'Inconsolata';
```

Comment:
That's there so our tables match table formatting on the blog. But I agree that it's not necessary here, especially since this content shouldn't be monospaced. I'll remove.